views/account/account_settings.php

```diff
@@ -72,11 +76,11 @@
                     <h4 class="modal-title" style="float: left;"><?= $firstname." ".$middleinit." ".$lastname ?></h4>
                     <div class="float-right" style="display: inline!important;">
                         <div class="float-right">
-                            <a href="/account-settings/modify/<?= implode('/', $params) ?>"><button class="btn btn-outline-secondary btn-sm">Edit</button></a>
+                            <?php if ($isSuperUser && hasRole(Role::Superuser) || !$isSuperUser && hasRole(Role::Admin)) { ?>
```

Comment:
Maybe put the two parts of the OR statement in parenthesis:
```php
if ( ($isSuperUser && hasRole(Role::Superuser))  ||  (!$isSuperUser && hasRole(Role::Admin)) )
```
I think this is still the logic you're looking for (and I think it still works how you have it, just separates it cleanly and is easier to read.

Reply:
Same logic, boolean operator precedence is Not -> And -> Or. Agree with parens!